x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/agent_config/details_page/index.tsx

```diff
@@ -3,58 +3,210 @@
+                  flush="left"
+                  size="xs"
+                >
+                  <FormattedMessage
+                    id="xpack.ingestManager.configDetails.viewAgentListTitle"
+                    defaultMessage="View all agent configurations"
+                  />
+                </EuiButtonEmpty>
+                <EuiTitle size="l">
+                  <h1>
+                    {(agentConfig && agentConfig.name) || (
+                      <FormattedMessage
+                        id="xpack.ingestManager.configDetails.configDetailsTitle"
+                        defaultMessage="Config '{id}'"
+                        values={{
+                          id: configId,
+                        }}
+                      />
+                    )}
+                  </h1>
+                </EuiTitle>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+            {agentConfig && agentConfig.description ? (
+              <Fragment>
+                <EuiSpacer size="s" />
+                <EuiText color="subdued" size="s">
+                  {agentConfig.description}
+                </EuiText>
+              </Fragment>
+            ) : null}
+          </EuiFlexItem>
+        </EuiFlexGroup>
+        <EuiSpacer size="l" />
+      </React.Fragment>
+    ),
+    [AGENT_CONFIG_LIST_URI, agentConfig, configId]
+  );
+
+  const headerRightContent = useMemo(
+    () => (
+      <EuiFlexGroup justifyContent={'flexEnd'} direction="row">
+        {[
+          {
+            label: i18n.translate('xpack.ingestManager.configDetails.summary.revision', {
+              defaultMessage: 'Revision',
+            }),
+            content: '999', // FIXME: implement version - see: https://github.com/elastic/kibana/issues/56750
+          },
+          {
+            label: i18n.translate('xpack.ingestManager.configDetails.summary.datasources', {
+              defaultMessage: 'Data sources',
+            }),
+            content: (
+              <EuiI18nNumber
+                value={
+                  (agentConfig && agentConfig.datasources && agentConfig.datasources.length) || 0
+                }
+              />
+            ),
+          },
+          {
+            label: i18n.translate('xpack.ingestManager.configDetails.summary.usedBy', {
+              defaultMessage: 'Used by',
+            }),
+            content: (
+              <LinkedAgentCount
+                count={(agentStatus && agentStatus.total) || 0}
+                agentConfigId={(agentConfig && agentConfig.id) || ''}
+              />
+            ),
+          },
+          {
+            label: i18n.translate('xpack.ingestManager.configDetails.summary.lastUpdated', {
+              defaultMessage: 'Last updated on',
+            }),
+            content:
+              (agentConfig && (
+                <FormattedDate
+                  value={agentConfig?.updated_on}
+                  year="numeric"
+                  month="short"
```

Comment:
@hbharding the mock for this date show a Long month, however, I implemented here with short month to match how it was displayed on the Agent Config list. Let me know if it should be long instead.

Reply:
Thanks for catching. Yes - lets use short month here!

Reply:
👍